code/modules/food_and_drink/plants.dm

```diff
@@ -11,8 +11,7 @@ ABSTRACT_TYPE(/obj/item/reagent/containers/food/snacks/plant)
 	var/crop_prefix = ""	// Prefix for crop name when harvested ("rainbow" melon)
 	var/crop_suffix = ""	// Suffix for crop name when harvested (bamboo "shoot")
 	food_effects = list("food_cold", "food_disease_resist")
-
-	var/made_reagents = 0
+	var/made_reagents = FALSE //set this to true so the stuff gets no new reagents added after creation
```

Comment:
doc comment

Reply:
To be honest, it was also a relic when i tried to overhaul the complete plants.dm . The variable isn't used anywhere else, so i assume it is safe to change. Should i revert this?

Reply:
No, just change the comment format to the documentation comment format